R/utils.R

```diff
@@ -90,23 +90,33 @@ h_free_cores <- function() {
 }
 
 #' covariance type
-#' 
+#'
 #' @format vector of supported covariance structures. `cov_type` for common time points covariance structures,
 #' `cov_type_spatial` for spatial covariance structures.
 #' @details
 #' abbreviation for covariance structures
 #' ## Common Covariance Structures
-#' * us: "unstructured"
-#' * toep: "Toeplitz"
-#' * toeph: "heterogeneous Toeplitz"
-#' * ar1: "auto-regressive order one"
-#' * ar1h: "heterogeneous auto-regressive order one"
-#' * ad: "ante-dependence"
-#' * adh: "heterogeneous ante-dependence"
-#' * cs: "compound symmetry"
-#' * csh: "heterogeneous compound symmetry"
+#'
```

Comment:
maybe good to have one sentence here explaining what `i` and `j` are (just to be very clear)